aap_chatbot/src/useChatbot/useChatbot.ts

```diff
@@ -222,16 +222,34 @@ export const useChatbot = () => {
       timestamp: getTimestamp(),
       referenced_documents: [],
     };
-    const sendFeedback = async (sentiment: Sentiment) => {
-      if (typeof response === "object") {
+    const sendFeedback = async (
+      sentiment: Sentiment,
+      content: string = "",
+      referenced_documents: ReferencedDocument[] = [],
+    ) => {
+      if (typeof response === "string") {
```

Comment:
On a non-streaming chat, `response` contains the response of the `/chat` API. To the contrary, on a streaming chat, `conversation_id` and `refrenced_douments` are sent in streaming data. For providing the equivalent data given to the feedback API in streaming chat, we need to build the same response as the one on non-streaming chat.